src/allencell_ml_segmenter/_tests/core/test_file_input_model.py

```diff
@@ -98,3 +99,120 @@ def test_set_max_channels_dispatch() -> None:
 
     # Assert nothing happened
     dummy_subscriber.was_handled(Event.ACTION_FILEINPUT_MAX_CHANNELS_SET)
+
+def test_get_input_files_as_list_from_path() -> None:
+    """
+    Test to see if all paths from a directory are returned as a list
+    """
+    # ARRANGE
+    file_input_model: FileInputModel = FileInputModel()
+    dummy_subscriber: FakeSubscriber = FakeSubscriber()
+    file_input_model.subscribe(
+        Event.ACTION_FILEINPUT_MAX_CHANNELS_SET,
+        dummy_subscriber,
+        dummy_subscriber.handle,
+    )
```

Comment:
Not sure I understand why we are using a fake subscriber for the max channels set event in this test

Reply:
I would expect the test to work just as well if you deleted lines 109-114?

Reply:
This comment applies to all of the new tests in this file--am I missing something?